sdk/dslink/src/main/java/org/dsa/iot/dslink/serializer/SerializationManager.java

```diff
@@ -25,6 +27,10 @@
     private final Serializer serializer;
     private ScheduledFuture<?> future;
 
+    private Cipher cipher;
```

Comment:
do you really need to store this as a field? Seems to me it's creating some complexity with the order of usage/initialization. I mean, sometimes you use the field, other times you assign it to a local variable. It's a bit confusing.